src/app-layout/drawer/interfaces.ts

```diff
@@ -83,3 +85,13 @@ export interface SizeControlProps {
   setSidePanelWidth: (width: number) => void;
   setBottomPanelHeight: (height: number) => void;
 }
+
+export interface InternalDrawerProps {
+  drawers: {
```

Comment:
This field should be optional, to reflect that the prop will not always be set. Typescript will then show some additional type errors in `app-layout/index.ts`

```suggestion
  drawers?: {
```